src/XRP/issued-currency-client.ts

```diff
@@ -329,25 +331,108 @@ export default class IssuedCurrencyClient {
     amount: string,
     wallet: Wallet,
   ): Promise<TransactionResult> {
-    if (!XrpUtils.isValidXAddress(issuerXAddress)) {
+    return await this.sendTrustSetTransaction(
+      issuerXAddress,
+      currencyName,
+      amount,
+      undefined,
+      wallet,
+    )
+  }
+
+  /**
+   * Creates an authorized trust line between this XRPL account and another account.
+   * An authorized trust line will only be created if `requiredAuthorizedTrustlines` is called first.
```

Comment:
what sucks is that if you DO NOT called `requireAuthorizedTrustlines` first, it will not error - no trust line will be created instead. as such, i *could* check to see if the correct account flag is set, but then we take on the additional cost of another fetch from the ledger.

i propose we wait until we are building out our demo to see how annoying this is in practice, but let me know what you think